server/chat-commands.js

```diff
@@ -4488,17 +4488,24 @@ const commands = {
 		let targetName = target;
 		let targetRegistered = false;
 		let targetToken = '';
+		let settings = {};
 		if (commaIndex >= 0) {
 			targetName = target.substr(0, commaIndex);
 			target = target.substr(commaIndex + 1);
 			commaIndex = target.indexOf(',');
 			targetRegistered = target;
 			if (commaIndex >= 0) {
 				targetRegistered = !!parseInt(target.substr(0, commaIndex));
-				targetToken = target.substr(commaIndex + 1);
+				target = target.substr(commaIndex + 1);
+				commaIndex = target.indexOf(',');
+				targetToken = target;
+				if (commaIndex >= 0) {
+					targetToken = target.substr(0, commaIndex);
+					settings = JSON.parse(target.substr(commaIndex + 1));
```

Comment:
Bit of a nitpick but this should be in a try ... catch block and you need to make sure it's an object so you don't crash when using `in` on it.